crates/solidity/testing/snapshots/cst_output/ConstructorDefinition/simple/generated/0.4.22.yml

```diff
@@ -0,0 +1,22 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+Source: >
+  1  │ constructor () {                                                                 │ 0..16
```

Comment:
are we now able to remove this comment from source?

```solidity
constructor () { }
```

Reply:
done